client/src/components/Disclaimer.js

```diff
@@ -4,25 +4,32 @@ import { ReactComponent as TextLogo } from '../img/Logo_CORONATRACKER_Text_Logo.
 import Button from 'react-bootstrap/Button';
 
 function Disclaimer(props) {
+  const [modalHidden, setModalHidden] = React.useState(true);
   return (
-    <div className="DisclaimerPopUp">
-      <div className="DisclaimerPopUpInner">
-        <Logo className="LoginLogo" className="PopUpLogo" />
-        <TextLogo className="TextLogo" className="PopUpLogo" />
-        <p>
-          The CoronaTracker is designed to help you navigate through the pandemic with accurate information, charting
-          your wellbeing, and tracking your health. The CoronaTracker is not intended to be used or viewed as diagnosis
-          or treatment of disease or other symptoms, including but not limited to COVID-19.
-        </p>
-        <p>
-          This application was made by the community for the community. This application is made for you! You are in
-          control of how you would like to contribute your data for public health and research.
-        </p>
-        <div>
-          <Button className="theme-Palette-white-text">I agree</Button>
-          <Button className="theme-Palette-white-text theme-Palette-red-bg">I don't agree</Button>
+    <div>
+      {modalHidden ? (
+        <div className="DisclaimerPopUp">
+          <div className="DisclaimerPopUpInner">
+            <Logo className="LoginLogo" className="PopUpLogo" />
+            <TextLogo className="TextLogo" className="PopUpLogo" />
+            <p>
+              The CoronaTracker is designed to help you navigate through the pandemic with accurate information,
+              charting your wellbeing, and tracking your health. The CoronaTracker is not intended to be used or viewed
+              as diagnosis or treatment of disease or other symptoms, including but not limited to COVID-19.
+            </p>
+            <p>
+              This application was made by the community for the community. This application is made for you! You are in
+              control of how you would like to contribute your data for public health and research.
+            </p>
+            <div>
+              <Button className="theme-Palette-white-text" onClick={() => setModalHidden(false)}>
+                I agree
+              </Button>
+              <Button className="theme-Palette-white-text theme-Palette-red-bg">I don't agree</Button>
+            </div>
+          </div>
         </div>
-      </div>
+      ) : null}
```

Comment:
While this will be needed in the future, we should try to keep PRs scoped to tackling one issue at a time.

We should remove this but then should definitely open up another issue addressing making `Disclimaer.js` stateful, and when we do, we'll want to use Redux as we should try and keep it as much of a SSOT as possible!

Reply:
Good point, I'll remove it for now and just add the fix for the route!

Reply:
I was typing haha! but @SomeMoosery. Also I wanted to point out that there is an issue [here](https://github.com/COVID-19-electronic-health-system/Corona-tracker/issues/188) and soon will be changed to material-ui